### DISCUSSION TITLE
`.chezmoiignore` works partially and confused as to why

### DISCUSSION THREAD

**Discussion by balaji-dutt** [twpayne/chezmoi#2917] [Q&A] [answered]
I have the following lines in my `.chezmoiignore` file:

~~~
{{ if ne .chezmoi.os "never" }}
.local/config/gitignore_global.txt
{{ end }}
README.md
.markdownlint.jsonc
{{ if ne .chezmoi.os "darwin" }}
.gitignore_global
{{ end }}

~~~

When I run `chezmoi ignored` I see the following output:

~~~sh
$ chezmoi ignored
.gitignore_global
.local/config/gitignore_global.txt
README.md
~~~

As you can see the file `.markdownlint.jsonc` does not appear in the output which would suggest that `chezmoi` would add the file to my `$HOME` directory. However, `chezmoi apply` does not actually create the file `.markdownlint.jsonc` in my `$HOME` directory. 

Can someone help me understand what I'm doing wrong here?

**Comment by bradenhilton** [comment, marked as answer]
https://www.chezmoi.io/reference/source-state-attributes/

> chezmoi ignores all files and directories in the source directory that begin with a `.` with the exception of files and directories that begin with `.chezmoi`.

**Reply by balaji-dutt** [reply]
Thank you! That cleared it up for me!
